data/item_data.json

```diff
@@ -4432,8 +4432,8 @@
 				"Blaze Rod": "Yes (unless in peaceful)",
 				"Ghast Tear": "Yes (unless in peaceful)",
 				"Potion": {
-					"All Except Healing": "Yes",
-					"Healing Potion": "Yes (unless in peaceful)"
+					"All": "Yes (unless in peaceful)",
+					"Healing<br>Fire Resistance<br>Weakness<br>Invisiblity": "Yes"
```

Comment:
I know it does not look too clean, but you should bear in mind the case of filtering for only values with "Yes (unless in peaceful)": potions will show up with only the label "all", and unless you already know the exceptions to that or choose to manually disable the filter to look for the second value then you won't know that "all" doesn't actually mean "all."